centreon/tests/php/Core/TimePeriod/Domain/Model/ExtraTimePeriodTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * Copyright 2005 - 2022 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Core\TimePeriod\Domain\Model;
+
+use Centreon\Domain\Common\Assertion\AssertionException;
+use Core\TimePeriod\Domain\Model\ExtraTimePeriod;
+use Core\TimePeriod\Domain\Model\TimeRange;
+
+$timeRange = new TimeRange('00:01-02:00');
+$dayRange = '';
```

Comment:
Isn't it better into a `beforeEach()` call ?

Reply:
No because the value changes depending on the test cases